servant-client/servant-client.cabal

```diff
@@ -46,9 +56,14 @@ library
     , text
     , transformers
     , transformers-compat
+    , case-insensitive
+  if impl(ghcjs)
+    build-depends:
+        ghcjs-base
+      , ghcjs-prim
   hs-source-dirs: src
   default-language: Haskell2010
-  ghc-options: -Wall
+  ghc-options: -Wall -Werror
```

Comment:
`-Werror` on travis but not the package seems more sensible (in fact I think hackage doesn't even let you upload packages with `Werror`).